### PR TITLE
Institution2

### PR DESCRIPTION
**Pull request by karleywalker**
Add Institution in Theme 4

### CODE REVIEW

File: public/themes/theme_4.json [hunk 1]
```diff
@@ -1,83 +1,105 @@
...
+                    "validation": null
+                  }
                 ]
-            }
+              }
+            ]
+          }
         },
         {
+<<<<<<< HEAD
```
Comment (Rytiggy) [inline public/themes/theme_4.json]: this is from a merge conflict and should not be here

File: public/themes/theme_4.json [hunk 2]
```diff
@@ -1,83 +1,105 @@
...
+              {
+                "type": "settings",
+                "items": [
+                  {
+                    "type": "checkbox",
+                    "label": "Show in navigation",
+                    "value": "showInNavigation",
+                    "validation": null
+                  }
+=======
```
Comment (Rytiggy) [inline public/themes/theme_4.json]: from a merge conflict and should not be here

File: public/themes/theme_4.json [hunk 3]
```diff
@@ -112,8 +134,124 @@
                         ],
                         "validation": null
                     }
+>>>>>>> 145337e976a6f84d2fa84d3d6800c42983e64c58
```
Comment (Rytiggy) [inline public/themes/theme_4.json]: from a merge conflict and needs to be removed

**Comment by caneruguz** [comment]
@karleywalker can you make a fake institution logo and use it for the top page?

File: public/themes/theme_4.json [hunk 4]
```diff
@@ -1,119 +1,220 @@
...
+            ]
+          }
+        },
+        {
+          "sectionHeader": "Image",
+          "component": "layer-image",
+          "settings": {
+            "values": {
+              "height": 500,
+              "url": "http://localhost:4200/img/building.jpg",
```
Comment (caneruguz) [inline public/themes/theme_4.json]: Is this the photo of German parliament? Can you note somewhere where you got it, if its unsplash you don't have to.

Comment (karleywalker) [inline public/themes/theme_4.json]: @caneruguz it is from unsplash. It was one of the only good "company/institution" looking buildings on there. I can make a fake institution logo too.

**Comment by karleywalker** [comment]
@Rytiggy

**Comment by Rytiggy** [comment]
looks good

**Pull request merged**